easylist/easylist_specific_hide.txt

```diff
@@ -1116,6 +1116,7 @@ computerworld.com,networkworld.com,probuildstats.com##.ad-placement
 comedy.com##.ad-placement-wrapper
 beincrypto.com##.ad-post-right-sidebar-1
 beincrypto.com##.ad-post-right-sidebar-2
+373news.com##.ad_response
```

Comment:
Need to be generic for two reason:
1. We don't add regional specific rule
2. This is not unique to the site, common in all Uzou recommendation